src/main/java/com/minecolonies/coremod/client/gui/WindowHireWorker.java

```diff
@@ -406,11 +400,15 @@ else if ((selectedModule.isFull()) && !selectedModule.getAssignedCitizens().cont
                 }
                 textBuilder.newLine(); // finish the current line
 
-                rowPane.findPaneOfTypeByID(CITIZEN_LABEL, Text.class)
-                  .setText((citizen.getJob().isEmpty() ? "" : LanguageHandler.format(citizen.getJob()) + ": ") + citizen.getName());
+                ITextComponent citizenLabelComponent =
+                  citizen.getJob().isEmpty() ? new StringTextComponent(citizen.getName()) : new TranslationTextComponent(citizen.getJob()).append(": ").append(citizen.getName());
+                rowPane.findPaneOfTypeByID(CITIZEN_LABEL, Text.class).setText(citizenLabelComponent);
                 rowPane.findPaneOfTypeByID(ATTRIBUTES_LABEL, Text.class).setText(textBuilder.getText());
                 final JobEntry entry = selectedModule.getJobEntry();
-                PaneBuilders.tooltipBuilder().hoverPane(rowPane.findPaneOfTypeByID(ATTRIBUTES_LABEL, Text.class)).build().setText(new TranslationTextComponent(entry.getKey().toString() + ".skills.desc"));
+                PaneBuilders.tooltipBuilder()
+                  .hoverPane(rowPane.findPaneOfTypeByID(ATTRIBUTES_LABEL, Text.class))
+                  .build()
+                  .setText(new TranslationTextComponent(entry.getKey().toString() + ".skills.desc"));
```

Comment:
could use append method in builder (i see it was there before and its not ur change but still)